src/resolvers/registries/npm-resolver.js

```diff
@@ -43,7 +43,7 @@ export default class NpmResolver extends RegistryResolver {
           'couldntFindVersionThatMatchesRange',
           body.name,
           range,
-          (versions.length > 20) ? versions.join(os.EOL) : versions.join(', '),
```

Comment:
@olingern thanks for the feedback. I'm going to close this PR anyway because there is more interesting work happening in #2136 that also solves this.

Cheers